table-api/src/main/java/io/deephaven/api/updateby/UpdateByOperation.java

```diff
@@ -1620,11 +1620,13 @@ static UpdateByOperation RollingCount(String timestampCol, long revTime, long fw
 
 
     /**
-     * Create a {@link RollingStdSpec rolling standard deviation} for the supplied column name pairs, using ticks as the
-     * windowing unit. Ticks are row counts and you may specify the previous window in number of rows to include. The
-     * current row is considered to belong to the reverse window, so calling this with {@code revTicks = 1} will simply
-     * return the current row. Specifying {@code revTicks = 10} will include the previous 9 rows to this one and this
-     * row for a total of 10 rows.
+     * Create a {@link RollingStdSpec rolling sample standard deviation} for the supplied column name pairs, using ticks
+     * as the windowing unit. Sample standard deviation is computed using Bessel's correction:
+     * https://en.wikipedia.org/wiki/Bessel%27s_correction This ensures that the sample variance will be an unbiased
+     * estimator of population variance. Ticks are row counts and you may specify the previous window in number of rows
+     * to include. The current row is considered to belong to the reverse window, so calling this with
+     * {@code revTicks = 1} will simply return the current row. Specifying {@code revTicks = 10} will include the
+     * previous 9 rows to this one and this row for a total of 10 rows.
```

Comment:
In all of these places, the bessel part should be a separate paragraph after the original paragraph.